Assets/Live2D/Cubism/Framework/MouthMovement/CubismMouthController.cs

```diff
@@ -42,7 +42,11 @@ public sealed class CubismMouthController : MonoBehaviour, ICubismUpdatable
         [HideInInspector]
         public bool HasUpdateController { get; set; }
 
-
+        /// <summary>
+        /// Blend Weight
+        /// </summary>
+        [SerializeField]
+        public float Weight = 1;
```

Comment:
```suggestion
        public float Weight = 1.0f;
```

We have suggested one modification to align with our coding conventions. Once that change is complete, we will be able to incorporate it.